pytorch_blade/src/compiler/mlir/converters/torch_mlir_op_filter.cpp

```diff
@@ -95,6 +96,8 @@ const std::unordered_set<std::string> &GetTorchMlirWhiteList() {
       "aten::silu",
       "aten::size",
       "aten::slice",
+      "aten::softmax",
```

Comment:
add softmax e2e ut in `build_pytorch_blade.sh#L63` ?

Reply:
done